ms-common/src/main/java/net/trajano/ms/core/JwtClaimsSetPrincipal.java

```diff
@@ -13,15 +16,44 @@
 public class JwtClaimsSetPrincipal implements
     Principal {
 
+    /**
+     * Subject claim key.
+     */
+    public static final String SUBJECT = "sub";
+
+    /**
+     * Issuer claim key.
+     */
+    public static final String ISSUER = "iss";
+
+    /**
+     * This is a framework specific claim which lists the roles in a String[].
+     */
+    public static final String ROLES = "roles";
+
     private final String authority;
 
-    private final JWTClaimsSet claimsSet;
+    private final String subject;
+
+    /**
+     * Roles.
+     */
+    private final Set<String> roles;
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename field "roles" to prevent any misunderstanding/clash with field "ROLES" defined on line 32. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1845)